pkgs/tools/filesystems/bees/default.nix

```diff
@@ -2,8 +2,8 @@
 
 let
 
-  version = "0.6.2";
-  sha256 = "05niv9rivd3j3cwcx3n3vjr85wr0l5i76giq9n54d2vdwmn8qjib";
+  version = "0.6.3";
+  hash = "sha256-brEjr7lhmKDCIDeLq+XP+ZTxv1RvwoUlszMSEYygxv8=";
```

Comment:
Any particular reason those two are outside stdenv.mkDerivation?

Reply:
I guess it's to have the "moving parts" together and at the file (it was like this before). What's your recommendation/preference?

Reply:
I added a second commit fixing this. I can squash the commits, if that's preferred.